middle_end/semantics_of_primitives.ml

```diff
@@ -142,6 +142,10 @@ let for_primitive (prim : Lambda.primitive) =
     Misc.fatal_errorf "The primitive %a should have been eliminated by the \
         [Closure_conversion] pass."
       Printlambda.primitive prim
+  | Pload8
+  | Pload16 _
+  | Pload _ ->
+    No_effects, Has_coeffects
```

Comment:
Shouldn't this be the same as for the string/bigstring primitives? i.e. `Arbitrary_effects, No_coeffects`